dotnet/src/webdriver/SeleniumManager.cs

```diff
@@ -0,0 +1,146 @@
+                    {
+                        throw new WebDriverException("Selenium Manager only supports Windows in .NET at this time");
+                    }
+
+                    try
+                    {
+                        string name = "selenium-manager-" + folder;
+                        using (Stream fileStream = ResourceUtilities.GetResourceStream(name, name))
+                        {
+                            using (BinaryReader binReader = new BinaryReader(fileStream, Encoding.ASCII))
+                            {
+                                byte[] fileBytes = binReader.ReadBytes((int)fileStream.Length);
+                                string directoryName = string.Format(CultureInfo.InvariantCulture, "webdriver.{0}",
+                                    Guid.NewGuid().ToString("N"));
+                                var path = Path.Combine(Path.GetTempPath(), directoryName + "/" + folder);
+                                Directory.CreateDirectory(path);
+                                var filePath = Path.Combine(path, "selenium-manager" + extension);
+
+                                using (BinaryWriter binWriter = new BinaryWriter(File.Open(filePath, FileMode.Create)))
+                                {
+                                    binWriter.Flush();
+                                    binWriter.Write(fileBytes);
+                                }
+                                binary = filePath;
+                            }
+                        }
+                    }
+                    catch (Exception ex)
+                    {
+                        throw new WebDriverException("Unable to obtain Selenium Manager", ex);
+                    }
+                }
+
+                return binary;
+            }
+        }
+
+        /// <summary>
+        /// Executes a process with the given arguments.
+        /// </summary>
+        /// <param name="fileName">The path to the Selenium Manager.</param>
+        /// <param name="arguments">The switches to be used by Selenium Manager.</param>
+        /// <returns>
+        /// the standard output of the execution.
+        /// </returns>
+        private static string RunCommand(string fileName, string arguments)
+        {
+            Process process = new Process();
+            process.StartInfo.FileName = fileName;
+            process.StartInfo.Arguments = arguments;
+            process.StartInfo.UseShellExecute = false;
+            process.StartInfo.RedirectStandardOutput = true;
+
+            try
+            {
+                process.Start();
+            }
+            catch (Exception ex)
+            {
+                throw new WebDriverException("Error starting process: " + process, ex);
+            }
+
+            String output = process.StandardOutput.ReadToEnd();
+
+            if (!output.StartsWith("INFO")) {
+                throw new WebDriverException("Invalid response from process: " + process);
```

Comment:
Include `output` into exception message to see what exactly the process wrote.

Reply:
right now output is always an empty string if there's a problem. If/when we start outputting more things (like errors), we can update.